### PR TITLE
Response enum property added removed checker CL entries

### PR DESCRIPTION
**Pull request by blva**
- Formalizing response enum property added/removed checker

### CODE REVIEW

File: checker/check-response-property-enum-value-removed_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tufin/oasdiff/checker"
+	"github.com/tufin/oasdiff/diff"
+)
+
+// CL: Adding an enum value to a response property
```
Comment (ciprian-tibulca) [inline checker/check-response-property-enum-value-removed_test.go]:
```suggestion
// CL: Removing an enum value from a response property
```

File: checker/check-response-property-enum-value-removed_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		Text:        "removed the 'TYPE2' enum value from the 'data/typeEnum' response property for the response status '200'",
+		Level:       checker.INFO,
+		Operation:   "POST",
+		Path:        "/api/v1.0/groups",
+		Source:      "../data/checker/response_property_enum_added_base.yaml",
+		OperationId: "createOneGroup",
+	}, errs[0])
+}
+
+// CL: Adding an enum value to a response write-only property
```
Comment (ciprian-tibulca) [inline checker/check-response-property-enum-value-removed_test.go]:
```suggestion
// CL: Removing an enum value from a response write-only property
```

**Review by ciprian-tibulca — APPROVED** [review]
LGTM

**Comment by reuvenharrison** [comment]
Hi @blva and @tibulca 
In the past I had permissions to commit on your PRs, now I don't.
Perhaps you can give me such access to speed up the merge process.
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

File: checker/localizations_src/en/messages.yaml [hunk 3]
```diff
@@ -153,6 +153,7 @@ response-property-became-required: the response property %s became required for
 response-write-only-property-became-required: the response write-only property %s became required for the status %s
 request-body-media-type-removed: removed the media type %s from the request body
 request-body-media-type-added: added the media type %s to the request body
+response-write-only-property-enum-value-added: added the new '%s' enum value the %s response write-only property for the response status %s
```
Comment (reuvenharrison) [inline checker/localizations_src/en/messages.yaml]: typo in:
added the new '%s' enum value the %s response write-only property for the response status %s
should be:
added the new '%s' enum value to the %s response write-only property for the response status %s

Comment (blva) [reply checker/localizations_src/en/messages.yaml]: thanks updated

**Comment by blva** [comment]
> allow

Hey @reuvenharrison that might not be possible given we're using MDB fork now and it has github actions. Sorry about that, but I'll address any comments you have.

File: checker/localizations_src/en/messages.yaml [hunk 4]
```diff
@@ -153,6 +153,7 @@ response-property-became-required: the response property %s became required for
 response-write-only-property-became-required: the response write-only property %s became required for the status %s
 request-body-media-type-removed: removed the media type %s from the request body
 request-body-media-type-added: added the media type %s to the request body
+response-write-only-property-enum-value-added: added the new '%s' enum value to the %s response write-only property for the response status %s
```
Comment (reuvenharrison) [inline checker/localizations_src/en/messages.yaml:156]: missing "to"

Comment (reuvenharrison) [inline checker/localizations_src/en/messages.yaml:156]: missing "to"

Comment (blva) [reply checker/localizations_src/en/messages.yaml:156]: Sorry I'm a bit confused as to where the "to" would be, perharps you can make a suggestion on the phrase? thanks!

Comment (reuvenharrison) [reply checker/localizations_src/en/messages.yaml:156]: looks ok now, I must have missed the fix

**Pull request merged**